xml/admin_monitoring_alerting.xml

```diff
@@ -189,717 +133,8 @@
   <para>
    The &alertmanager; handles alerts sent by the &prometheus; server. It takes
```

Comment:
I'd tend to say Prometheus fires alerts and Alertmanager sends them out. The Ceph Dashboards will automatically be configured to receive them. Technically it is possible to configure the alertmanager to send alerts to other services, but we haven't documented that anywhere yet and with cephadm, it has also not yet been tested. Cephadm overwrites the alertmanagers configuration, which means I'd need to investigate how that can be done before we can think of documenting it.

Reply:
I'm going to leave this for now then and add a note to the docs